worldedit-sponge/src/main/java/com/sk89q/worldedit/sponge/SpongeWorldEdit.java

```diff
@@ -31,6 +31,7 @@
 import com.sk89q.worldedit.sponge.config.SpongeConfiguration;
 import com.sk89q.worldedit.sponge.nms.NMSHelper;
 import com.sk89q.worldedit.sponge.nms.SpongeNMSWorld;
+import org.apache.logging.log4j.Logger;
```

Comment:
Docs actually say `org.slf4j.Logger` -- not sure if this `Logger` would work, though it *is* the right implementation class...